### DISCUSSION TITLE
I choose any network will appear this problem!

### DISCUSSION THREAD

**Discussion by Ed1s0nZ** [Security-Onion-Solutions/securityonion#5160] [Unsupported Versions]
The IP being routed by Linux is not the IP address assigned to the management interface (eno1).                                            
This is not a supported configuration, please remediate and rerun

I choose any network will appear this problem!
![image](https://user-images.githubusercontent.com/83812544/129527989-c677cfef-e084-434a-9f16-4934a70aa7fa.png)

![image](https://user-images.githubusercontent.com/83812544/129527877-aef6d14c-6c5b-4b1a-ba7e-386962704a73.png)

**Comment by weslambert** [comment]
Are you using a special/bonded configuration for the management interface?

**Reply by Ed1s0nZ** [reply]
I don't know. I didn't do anything ...TAT

**Comment by romarroca** [comment]
I am also having this error. I'm using vmware workstation.
